src/lib.rs

```diff
@@ -186,7 +186,7 @@ pub enum PandocOption {
     /// --reference-odt=FILENAME
     ReferenceOdt(PathBuf),
     /// --reference-docx=FILENAME
-    ReferenceDocx(PathBuf),
+    ReferenceDoc(PathBuf),
```

Comment:
Since this is a breaking change, could you instead add the new variant in addition to the existing one and add deprecation attributes on the old one?

Reply:
Yes, no problem. Something like this?

Reply:
Exactly like this, thanks!